modules/schema/src/main/java/org/apache/ignite/internal/schema/BinaryTuplePrefix.java

```diff
@@ -51,13 +52,36 @@ public BinaryTuplePrefix(int elementCount, ByteBuffer buffer) {
         super(elementCount, buffer);
     }
 
+    /**
+     * Creates a prefix from provided {@link BinaryTuple}. If given tuple has lesser or equal number of
+     * columns, then all elements will be used in resulting prefix. If given tuple has more columns, then
+     * excess columns will be truncated.
+     *
+     * @param size The size of the complete tuple.
```

Comment:
renamed `size` to `numElements`.  But renaming of entire method doesn't seem correct since it not only truncates given tuple, but may expand it as well